linux/libkmfl/debian/gbp.conf

```diff
@@ -0,0 +1,9 @@
+[DEFAULT]
+upstream-branch = upstream
+debian-branch = master
```

Comment:
Does this mean debian will track the `master` branch?

Reply:
no, this defines just the names of the branches where the import happens on the Debian side. However, these are the values that we'd get anyways, and I agree that this can be confusing, so I'll remove those values.